.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @joram @dlsteuer
+* @brandonb927 @dlsteuer
```

Comment:
@joram and I talked about this, he is fine handing over the reigns for Play to me